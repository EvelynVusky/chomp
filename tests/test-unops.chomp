int main() {
    nibble b = {{0101}};
    byte c = {{11100010}};
    word d = {{0111000110011110}};
    int i = 401932;

    /* binnot */
    println(~{{1}});
    println(~b);
    println(~c);
    println(~d);

    /* neg */
    println(-i);
    println(-13013);

    return 0;
}