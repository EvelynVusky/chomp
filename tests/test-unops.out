0
1010
00011101
1000111001100001
-401932
-13013
